### ISSUE TITLE
A question about  func 'build_opt_lr' when start train backbone

### ISSUE THREAD

**Issue by dypromise** [STVIR/pysot#145]
Hi, authors! 
A question confused me, when global epoch is up to cfg.BACKBONE.TRAIN_EPOCH, train code will build a new optimizer and lr_scheduler, and what confused me is that all trained_parameters' lr ARE SET back to origin setting(cfg.TRAIN.BASE_LR), and at the end of func 'build_opt_lr': lr_scheduler.step(cfg.TRAIN.START_EPOCH) will set all params learning rate back to beginning.  Is there anything wrong of what I think? why not set the learning rate of parameters to continue the learning rate from last epoth?  Thanks for any reply!

**Comment by lb1100** [comment]
we use `lr_scheduler` to control current learning rate. See [train.py](https://github.com/STVIR/pysot/blob/master/tools/train.py#L114)
